NOTICES

```diff
@@ -2411,19 +2411,23 @@ APACHE 2.0 LICENSED CODE:  The Program includes all or portions of
 the following software which IBM obtained under the terms and conditions
 of the Apache License Version 2.0:
 
-api version 0.21.2
-apimacherniery version 0.21.2
-client-go version 0.21.2
+apiextensions-apiserver version 0.18.0
+api version 0.18.0
+apimacherniery version 0.18.0
+cloud-provider version 0.18.0
+client-go version 0.18.0
 code-generator-kubernetes version 1.13.1
-controller-runtime version 0.9.2
+controller-runtime version 0.6.2
 controller-tools version 0.1.11
-controller-util version 0.2.6
+controller-util commit fc080f9
 gengo commit e17681d
+grpc version 1.27.0
+kube-openapi version v0.0.0-20200121204235-bf4fb3bd569c
 logr version 0.1.0
-operator-sdk version 1.9.0
-yaml version 1.2.0
+operator-sdk version 0.18.1

```

Comment:
why were all the dependencies downgraded here?

Reply:
IDK, its 1.9.0 in the OSSC

Reply:
I will raise it to legal department

Reply:
thanks